notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -120,7 +120,7 @@
    "outputs": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    fig.set_figheight(fig.get_figheight() * 0.7)
</pre><p>Nit: consider defining 0.7 as a constant somewhere since it's used in two places and we want to make sure it's consistent between the figures.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>